src/stacks-hierarchy/Root_TabNavigatorStack/TabNav_ProfileStack/Profile_NotificationsScreen.tsx

```diff
@@ -117,6 +153,29 @@ export const Profile_NotificationsScreen = () => {
               )}
             />
           )}
+          <ContentHeading
+            text={t(
+              ProfileTexts.sections.settings.linkSectionItems.notifications
+                .groupsHeading,
+            )}
+          />
+          <Section>
+            {notificationConfig?.groups.map((group) => (
+              <ToggleSectionItem
+                key={group.id}
+                text={group.toggleTitle[0].value ?? ''}
```

Comment:
These language props should use `getTextForLanguage`, e.g. `getTextForLanguage(group.toggleTitle, language)`. 

I suppose `toggleTitle` is always defined in practice, but in theory it could be `undefined`, and `getTextForLanguage` is typed to potentially return undefined. The `?? ''` is therefore necessary. If the `toggleTitle` for any stange reason actually is undefined I think it makes little sense to show this toggle. I think this toggle should be shown only if `getTextForLanguage(group.toggleTitle, language)` actually returns  a value. If you decide to do that I think the code will be more readable if this toggle section item is extracted to a separate component.

Reply:
Oups, this is some remains from a dirty first draft 🙈 Will fix! Can turn it into a component as well. We should support an empty toggleDescription, but probably not toggleTitle, so I can hide it in that case.

Reply:
Added a `.filter` and proper getTextForLanguage now. Making a separate component got quite messy, especially since the `<Section>` doesn't like it when you give it non-section-item-components.